### ISSUE TITLE
Propagate -parallelism=50 tf plan CLI argument to planner.go

### ISSUE THREAD

**Issue by vladgolubev** [mbode/terraform-state-mover#25]
To achieve faster plan, I had to modify source code to make 22nd line of `planner.go` file like the following:
```go
if err := terraformExec(cfg, true, args, "plan", "-out="+tfPlanName, "-parallelism=60"); err != nil {
```

Would be nice to have this as a built-in feature

**Comment by mbode** [comment]
Thank you for pointing that out! This should always have been possible using `terraform-state-mover -- -parallelism=60`, but the passing of extra arguments to `terraform plan` turned out to be broken. Should be fixed now and I have also added an explicit example involving `-parallelism` to the README.

**Comment by vladgolubev** [comment]
Thank you very much, @mbode!

Your project is a huge time saver! There are no excuses to avoid refactoring my terraform code now ;)

**Comment by mbode** [comment]
Glad to hear that @vladgolubev ☺️

The fixed version has now been released as [0.5.0](https://github.com/mbode/terraform-state-mover/releases/tag/0.5.0).
